main/EDTypesMod.F90

```diff
@@ -4,7 +4,6 @@ module EDTypesMod
   use FatesConstantsMod,     only : ifalse
   use FatesConstantsMod,     only : itrue
   use FatesGlobals,          only : fates_log
-  use shr_infnan_mod,        only : nan => shr_infnan_nan, assignment(=)
```

Comment:
Looks like this wasn't actually being used?

Reply:
yup, not used